### PR TITLE
Feat/Staking view portfolio bar

### PR DESCRIPTION
**Pull request by mrcnk**
# Checklist

- [x]
- [ ] Proper tests implemented
- [x] Screenshots added.

---

## Proposed solution

- Add portfolio bar to represent current state of portfolio building.

## Screenshots

image
image

### CODE REVIEW

File: packages/core/src/ui/assets/icons/arrow-right.component.svg
Comment (przemyslaw-wlodek) [inline packages/core/src/ui/assets/icons/arrow-right.component.svg:1]: Isn't it used by `SocialLink` component?

File: packages/staking/src/features/staking/PortfolioBar.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  const { t } = useTranslation();
+  return (
+    <Card.Elevated className={barStyles}>
+      <Text.Body.Normal>
+        <span className={spanBold}>{t('portfolioBar.selectedPools', { selectedPoolsCount: 1 })}</span>&nbsp;
+        <span>{t('portfolioBar.maxPools', { maxPoolsCount: 5 })}</span>
+      </Text.Body.Normal>
+      <Flex className={sx({ gap: '$16' })}>
+        <Button.Secondary label="Clear" />
+        <Button.Primary label="Next" icon={<img src={String(ArrowRightIcon)} alt="Go next" />} />
```
Comment (przemyslaw-wlodek) [inline packages/staking/src/features/staking/PortfolioBar.tsx:16]: SVGs with `.component.svg` should be automatically converted to React components using SVGO. Please let's fix svgo config if needed (we have it in place). If that's too complex for this PR then let's create a ticket for that.

File: packages/staking/src/features/staking/PortfolioBar.css.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { sx } from '@lace/ui';
+
+export const barStyles = sx({
+  alignItems: 'center',
+  bottom: '$24',
+  display: 'flex',
+  justifyContent: 'space-between',
+  left: '$0',
+  padding: '$16',
+  position: '$sticky',
```
Comment (przemyslaw-wlodek) [inline packages/staking/src/features/staking/PortfolioBar.css.ts:10]: Shouldn't we use `fixed` here? I don't see the benefit of using `sticky`

**Review by przemyslaw-wlodek — CHANGES_REQUESTED** [review]

File: packages/core/src/ui/assets/icons/arrow-right.component.svg
Comment (mrcnk) [reply packages/core/src/ui/assets/icons/arrow-right.component.svg:1]: image

validated and the social links are fine

**Pull request closed without merging**